examples/sponsoredTXs/src/utils.ts

```diff
@@ -0,0 +1,346 @@
+        } as unknown as UpdateContractPayload,
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        {
+            owner: { Account: [account] },
+        },
+        SPONSORED_TX_RAW_SCHEMA
+    );
+}
+
+/**
+ * Action for submitting an transfer sponsored transaction to the cis3_nft smart contract instance.
+ */
+
+export async function submitTransferSponsoredTx(
+    connection: WalletConnection,
+    account: string,
+    signer: string,
+    nonce: string,
+    signature: string,
+    tokenID: string,
+    from: string,
+    to: string
+) {
+    if (nonce === undefined || nonce === '') {
+        // eslint-disable-next-line no-alert
+        alert('Insert a nonce.');
+        return '';
+    }
+
+    // eslint-disable-next-line no-restricted-globals
+    if (isNaN(Number(nonce))) {
+        // eslint-disable-next-line no-alert
+        alert('Your nonce needs to be a number.');
+        return '';
+    }
+
+    if (tokenID === undefined || tokenID === '') {
+        // eslint-disable-next-line no-alert
+        alert('Insert a tokenID.');
+        return '';
+    }
+
+    if (tokenID.length !== 8) {
+        // eslint-disable-next-line no-alert
+        alert('TokenID needs to have 8 digits.');
+        return '';
+    }
+
+    if (signature === undefined || signature === '') {
+        // eslint-disable-next-line no-alert
+        alert('Insert a signature.');
+        return '';
+    }
+
+    if (signature.length !== 128) {
+        // eslint-disable-next-line no-alert
+        alert('Signature needs to have 128 digits.');
+        return '';
+    }
+
+    if (to === undefined || to === '') {
+        // eslint-disable-next-line no-alert
+        alert('Insert an `to` address.');
+        return '';
+    }
+
+    if (to.length !== 50) {
+        // eslint-disable-next-line no-alert
+        alert('`To` address needs to have 50 digits.');
+        return '';
+    }
+
+    if (from === undefined || from === '') {
+        // eslint-disable-next-line no-alert
+        alert('Insert an `from` address.');
+        return '';
+    }
+
+    if (from.length !== 50) {
+        // eslint-disable-next-line no-alert
+        alert('`From` address needs to have 50 digits.');
+        return '';
+    }
+
+    if (signer === undefined || signer === '') {
+        // eslint-disable-next-line no-alert
+        alert('Insert an signer address.');
+        return '';
+    }
+
+    if (signer.length !== 50) {
+        // eslint-disable-next-line no-alert
+        alert('Signer address needs to have 50 digits.');
+        return '';
+    }
+
+    const message = {
+        contract_address: {
+            index: Number(SPONSORED_TX_CONTRACT_INDEX),
+            subindex: 0,
+        },
+        entry_point: 'contract_transfer',
+        nonce: Number(nonce),
+        payload: {
+            Transfer: [
+                [
+                    {
+                        amount: '1',
```

Comment:
What is the meaning of this hardcoded amount?

Reply:
The smart contract is an NFT smart contract. So very every `token_ID` there exists exactly one token. Transfer this token means that the amount is always 1.